src/views/Releases/ListReleases/index.jsx

```diff
@@ -128,22 +155,47 @@ function ListReleases(props) {
             const release = clone(r);
 
             if (sc) {
-              release.scheduledChange = sc;
-              release.scheduledChange.when = new Date(
-                release.scheduledChange.when
-              );
+              release.scheduledChange = buildScheduledChange(sc);
             }
 
-            // todo: set these
-            release.required_signoffs = {};
-
             return release;
           })
         );
       }
     );
   }, []);
 
+  useEffect(() => {
+    const productsReleases = releases.reduce((acc, release) => {
+      const pr = acc;
+
+      if (!requiresSignoffs(release)) {
+        pr[release.product] = release.name;
+      }
+
+      return pr;
+    }, {});
```

Comment:
This is because if the release requires signoffs, it is not necessary get `required signoffs for product`.

Reply:
Ah, because they're included in the returned data already? This is a worth a comment, as its not at all obvious (even for someone like me who should know!).